modelql-client/src/jvmTest/kotlin/org/modelix/modelql/client/ModelQLClientTest.kt

```diff
@@ -279,4 +283,15 @@ class ModelQLClientTest {
         }
         assertEquals(3, result)
     }
+
+    @Test
+    fun `resolving a non-existing node reference returns 404`() = runTest { httpClient ->
```

Comment:
We should probably document that explicitly in the OpenAPI specification.

Reply:
Updated the OpenAPI spec.